### PR TITLE
Updated to add JaCoCo test reports to Jenkins CI

### PR DESCRIPTION
**Pull request by fewyn**
Updated to add JaCoCo test reports to Jenkins CI.

This will also fail the build if coverage drops more than 5% which is adjustable via the jenkinsfile.

### CODE REVIEW

**Comment by fewyn** [comment]
Already tested and running on Jenkins once it's merged I'll switch it back to using the jenkinsfile.

File: jenkinsfile [hunk 1]
```diff
@@ -60,4 +47,4 @@ pipeline {
       }
     }
   }
-}
+}
```
Comment (heeheehee-kolmafia) [inline jenkinsfile:50]: nit: can you add a newline at the end of this file?

Otherwise this change looks fine to me.

Comment (fewyn) [reply jenkinsfile]: Done.

**Review by heeheehee-kolmafia — APPROVED** [review]
Thanks, looks good.

**Pull request merged**